app/controllers/catalog_controller.rb

```diff
@@ -544,9 +544,10 @@ def svc_catalog_provision
       options[:target_id] = st.id
       options[:target_kls] = st.class.name
       options[:dialog_locals] = {
-        :api_submit_endpoint => "/api/service_catalogs/#{st.service_template_catalog_id}/service_templates/#{st.id}",
-        :api_action          => "order",
-        :cancel_endpoint     => "/catalog/explorer"
+        :api_submit_endpoint    => "/api/service_catalogs/#{st.service_template_catalog_id}/service_templates/#{st.id}",
+        :api_action             => "order",
+        :finish_submit_endpoint => "/miq_request/show_list",
```

Comment:
@eclarizio i am wondering if we need a role_allows check here, what if user does not have access to Request list view, for reference check here to see how old code was handling this: https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller/dialog_runner.rb#L50

Reply:
You're probably right, I didn't realize that screen was access restricted.